Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyAudioCtrl.h

```diff
@@ -29,6 +29,27 @@ class QMimeData;
 
 namespace AzToolsFramework
 {
+    //=============================================================================
+    // Audio Control Selector Request Bus
+    // For connecting UI proper
+    //=============================================================================
+    class AudioControlSelectorRequests
+        : public AZ::EBusTraits
+    {
+    public:
+        // EBusTraits
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
+        using BusIdType = AudioPropertyType;
+
+        virtual AZStd::string SelectResource(const AZStd::string& previousValue)
+        {
+            return previousValue;
+        }
```

Comment:
I don't understand what this funciton does?
It's called `SelectResource`, but the input parameter is previousValue.
What does previousValue mean in this context?

Also because this function returns a copy of an AZStd::string, the input parameter should be an `AZStd::string_view` instead of a `const AZStd::string&`
```suggestion
        virtual AZStd::string SelectResource(AZStd::string_view previousValue)
        {
            return previousValue;
        }
```